client/my-sites/plugins/plugin-meta/index.jsx

```diff
@@ -436,7 +436,7 @@ class PluginMeta extends Component {
 		} );
 
 		const plugin = this.props.selectedSite && this.props.sites[ 0 ] ? this.props.sites[ 0 ].plugin : this.props.plugin;
-		const path = this.getExtensionSettingsPath( plugin );
+		const path = ( ! this.props.selectedSite || this.props.isInstalledOnSite ) && this.getExtensionSettingsPath( plugin );
```

Comment:
Addressed in [0306974](https://github.com/Automattic/wp-calypso/pull/16222/commits/0306974b364f4b9df5ee6a9e5861e6a35971178b)